server/src/test/java/org/elasticsearch/indices/recovery/DanglingIndicesIT.java

```diff
@@ -51,21 +51,23 @@ public void testDanglingIndicesAreRecoveredWhenSettingIsEnabled() throws Excepti
         final Settings settings = buildSettings(true);
         internalCluster().startNodes(3, settings);
 
-        execute("create table test(id integer) clustered into 2 shards with(number_of_replicas = 2)");
+        execute("create table doc.test(id integer) clustered into 2 shards with(number_of_replicas = 2)");
+        ensureGreen("test");
 
```

Comment:
The original adds a version for deleting a closed index, but we cannot deleted closed tables. Therefore it is skipped.
https://github.com/elastic/elasticsearch/commit/40801af88406ad130904940604407a0191c0d92e#diff-f16f8a255f9aae1f40edadf1e50857b30210605163d1be268042db90af74381fR56